mne/source_estimate.py

```diff
@@ -1370,18 +1370,14 @@ def plot(self, subject=None, surface='inflated', hemi='lh',
         hemi : str, 'lh' | 'rh' | 'split' | 'both'
             The hemisphere to display. Using 'both' or 'split' requires
             PySurfer version 0.4 or above.
-        colormap : str
-            The type of colormap to use.
+        colormap : str | array
```

Comment:
There are a number of options... Something like this okay though?
`colormap : str | np.ndarray of int or float, shape(n_colors, 3 | 4)`